src/renderer/Core/Search/SearchViewController.ts

```diff
@@ -67,6 +67,10 @@ export const useSearchViewController = ({
 
     const { value: fuzziness } = useSetting({ key: "searchEngine.fuzziness", defaultValue: 0.5 });
     const { value: maxSearchResultItems } = useSetting({ key: "searchEngine.maxResultLength", defaultValue: 50 });
+    const { value: maxSearchResultItemsEmptySearchTerm } = useSetting({
+        key: "searchEngine.maxResultLengthEmptySearchTerm",
+        defaultValue: 50,
```

Comment:
The default of 50 keeps behavior unchanged for users who prefer the current behavior, to keep the blast radius of this change minimal.